pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -98,6 +105,10 @@ export default Vue.extend<Data, Methods, Computed, never>({
           v-clean-html="t(`snapshots.info.${ snapshotEvent.type }.${ snapshotEvent.result }`, { snapshot: snapshotEvent.snapshotName, error: snapshotEvent.error }, true)"
           class="event-message"
         />
+        <span
+          v-clean-html="t('snapshots.info.when', { when: showWhen() }, true)"
```

Comment:
Why does this need to be raw? `YYYY-MM-DD HH:mm` should not produce any HTML.

Reply:
Cargo cult programming. I didn't know where `t` is defined, and it's kind of hard to search. So where is it defined anyway?

Reply:
https://github.com/rancher-sandbox/rancher-desktop/blob/7848d722b0d6a2a15a8f3953cc527c4681b6d865/pkg/rancher-desktop/plugins/i18n.js#L28-L30

Every time I look though I have read up on the definition of `stringFor` (in the same file) to see what it does.